stock_warehouse_flow_packaging_type/models/stock_warehouse_flow.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class StockWarehouseFlow(models.Model):
+    _inherit = "stock.warehouse.flow"
+
+    packaging_type_ids = fields.Many2many(
+        "product.packaging.type", string="Packaging types"
+    )
+    split_method = fields.Selection(
+        selection_add=[("packaging_type", "By Packaging Type")]
```

Comment:
The model changed to packaging level in v16 https://github.com/OCA/product-attribute/pull/1215, so I would make the key agnostic of this
```suggestion
        selection_add=[("packaging", "By Product Packaging")]
```